src/actionlib/action_client.py

```diff
@@ -460,6 +480,8 @@ def init_goal(self, goal, transition_cb=None, feedback_cb=None):
         csm = CommStateMachine(action_goal, transition_cb, feedback_cb,
                                self.send_goal_fn, self.cancel_fn)
 
+        csm.set_lost_goal_timestamp_delta(self.get_lost_goal_timestamp_delta())
```

Comment:
As suggested above, this would become:  `csm.lost_goal_delta = self.lost_goal_delta`